src/components/Navrail/Navrail.module.scss

```
@@ -0,0 +1,22 @@
+.Menu {
+  padding: 8px;
+  margin-top: 16px;
```

you can rewrite the margin styling like so:
```suggestion
  margin: 16px 8px;
```

you shouldn't set the bottom margin to 0 since there should always be a gap between the bottom and the separator